torch/csrc/distributed/c10d/init.cpp

```diff
@@ -26,13 +26,16 @@
 #include <pybind11/chrono.h>
 
 #include <c10d/comm.hpp>
+#include <c10d/frontend.hpp>
 #include <c10d/reducer.hpp>
 #include <torch/csrc/Exceptions.h>
 #include <torch/csrc/distributed/c10d/python_comm_hook.h>
 #include <torch/csrc/jit/python/pybind_utils.h>
 #include <torch/csrc/utils/object_ptr.h>
 #include <torch/csrc/utils/pybind.h>
 
+#include <torch/custom_class.h>
```

Comment:
why would we need this header? seems like we don't need it.

Reply:
It contains definition for `torch::class_`